lib/templates/deploy.bash.ruby.template

```diff
@@ -33,19 +53,32 @@ echo "$DEPLOYMENT_TARGET"
 if [ -e "$DEPLOYMENT_TARGET/Gemfile" ]; then
   echo "Found gemfile"
   pushd "$DEPLOYMENT_TARGET"
+  
   eval "$(rbenv init -)"
   exitWithMessageOnError "init failed"
+  
   echo "Setting ruby version"
-  rbenv global $WEBSITES_LATEST_NODE_VERSION
+  rbenv global $LATEST_RUBY_VERSION
   exitWithMessageOnError "Failed to switch ruby versions"
-  bundle config --local path "vendor/bundle"
+  
   echo "Running bundle clean"
   bundle clean --force
+  
   echo "Running bundle install"
-  bundle install --path "vendor/bundle" $OPTIONS
-  echo "Running bundle package"
-  bundle package --all
+  mkdir -p /tmp/bundle
+  bundle config --global path $BUNDLE_INSTALL_LOCATION
+  bundle install --no-deployment $OPTIONS
   exitWithMessageOnError "bundler failed"
+  
+  echo "Zipping up bundle contents"
+  tar -zcf /tmp/gems.tgz -C $BUNDLE_INSTALL_LOCATION .
+  exitWithMessageOnError "Error compressing gems"
+  
+  mkdir -p $SITE_CONFIG_DIR
+  exitWithMessageOnError "Error making config directory"
+  
+  cp -f /tmp/gems.tgz $SITE_CONFIG_DIR
```

Comment:
Is there no decompress that has to be done? What ends up happening if target dir already exists with stuff in it?